api/nakadi-event-bus-api.yaml

```diff
@@ -606,22 +488,394 @@ paths:
+        between streams of one subscription. The minimal consumption unit is a partition, so it is possible to start as
+        many streams as the total number of partitions in event-types of this subscription. The rebalance currently
+        only operates with the number of partitions so the amount of data in event-types/partitions is not considered
+        during autorebalance.
+        The position of the consumption is managed by Nakadi. The client is required to commit the cursors he gets in
+        a stream. This also depends on the `commit_mode` parameter that is set for the stream.
+      parameters:
+        - $ref: '#/parameters/SubscriptionId'
+        - name: commit_timeout
+          in: query
+          description: |
+            The amount of time allowed between commits. If no commit is performed within this amount of time
+            - the connection will be closed from Nakadi side. Specified in seconds.
+          type: integer
+          format: int32
+          required: false
+          default: 30
+        - name: window_size
+          in: query
+          description: |
+            The amount of uncommitted events Nakadi will stream before pausing the stream. When in paused
+            state and commit comes - the stream will continue. Minimal window size is 1.
+          type: integer
+          format: int32
+          required: false
+          default: 1000
+        - name: commit_mode
+          in: query
+          description: |
+            The commit mode that will be used for this stream. Currently only `commit_at_will` is supported.
+            Possible values:
+
+            - `commit_at_will`: client is free to choose how often he wants to commit. Of course the commit_timeout
+            property should still be respected. When a batch is committed that also automatically commits all previous
+            batches that were sent in a stream for this partition.
+
+            - `commit_each`: this commit mode requires each batch to be committed, and it should be done in correct
+            order. Committing a cursor when the previous is not yet committed is not possible.
+
+            - `auto_commit`: everything that Nakadi sends to stream is considered to be committed. No actions are
+            required from client side to commit. Be aware that with this commit mode loosing of some events is very
+            probable. So you should know what you are doing if you use autocommit.
+          type: string
+          enum:
+            - commit_at_will
+            - commit_each
+            - auto_commit
+          required: false
+          default: commit_at_will
+        - $ref: '#/parameters/BatchLimit'
+        - $ref: '#/parameters/StreamLimit'
+        - $ref: '#/parameters/BatchFlushTimeout'
+        - $ref: '#/parameters/StreamTimeout'
+        - $ref: '#/parameters/StreamKeepAliveLimit'
+        - $ref: '#/parameters/XFlowId'
+      responses:
```

Comment:
Yes, I think we can add that. But I'm not really sure in which exact case we should return it.